pkg/ccl/changefeedccl/parallel_io.go

```diff
@@ -84,15 +84,18 @@ func NewParallelIO(
 	metrics metricsRecorder,
 	settings *cluster.Settings,
 ) *ParallelIO {
+	quota := uint64(requestQuota.Get(&settings.SV))
```

Comment:
I would like to see us add a limit to this quota. If not to the setting itself, then at least here -- to set it to min(128, requestQuota.Get())

Reply:
I also would like to see this parameter become metamorphic.